lang/fr/texts/index-pro.html

```diff
@@ -35,6 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><b>Nouveau!</b>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
+
+<p>L'Éco-score arrive bientôt ! Le moteur d'Open Food Facts utilise la formule officielle de l'Éco-score.</p>
+  <ul>
+	  <li>Renseignez les données permetttant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
```

Comment:
```suggestion
	  <li>Renseignez les données permettant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
```

Same here: it's very confusing if we don't say which data is needed.